java/src/jmri/jmrix/nce/NceConnectionStatus.java

```diff
@@ -123,6 +124,15 @@ public NceMessage nceEpromPoll() {
             }
         }
 
+        // determine if really connected to command station by issuing dummy locomotive
+        // command, to short address 0.
+        if (epromState == CHECK_STATE) {
+            if (tc.getCommandOptions() > NceTrafficController.OPTION_2004) {
+                return NceMessage.sendLocoCmd(tc, 0x0000, NceBinaryCommand.LOCO_CMD_NOP, (byte) 00);
```

Comment:
This throws an "ncemon.NceMonBinary ERROR - Unhandled loco cmd op1 code: 0" message in the log if the connection is broken and restored,

We should modify the case statement at line 156 of NceMonBinary.java to show "Select Loco".